### ISSUE TITLE
No authorization to deploy 

### ISSUE THREAD

**Issue by Sar4D** [Azure/SimuLand#25]
Getting permissions problems when trying to deploy the AD FS

Keep getting the error ''You don’t have authorization to perform action 'Microsoft.Resources/deployments/validate/action'.''
Tried with both global administrator and root account. Tried to toggle to yes in the AD properties Access management for Azure resources .
Has anyone encounter that problem before? 
![Capture](https://user-images.githubusercontent.com/100158491/157113559-c96e0a6b-98c5-448f-b423-ef5266dd804a.PNG)

**Comment by Cyb3rWard0g** [comment]
Hello @Sar4D ! You need to log out and log back in. When I was using azure CLI, I had to log out and `az login` again. The template is of tenant scope and the easiest way is to do it with a privileged account.

**Comment by Cyb3rWard0g** [comment]
I tested the instructions in here and it worked fine:

https://github.com/Azure/SimuLand/tree/main/2_deploy/aadHybridIdentityADFS#elevate-account-access

I tried the first step only which is in the Azure Portal, and I got the error above in your message. However, after running step 2 via Azure CLI and logging out and logging back in, I was able to deploy the `TenantScope` template.

Additional References in case you want to know how to do it with PowerShell too. Same instructions.

https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-Setup-azure.md

Follow steps:
1- https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-Setup-azure.md#1-elevate-access-to-manage-azure-resources-in-the-directory
2- https://github.com/Azure/Enterprise-Scale/blob/main/docs/EnterpriseScale-Setup-azure.md#2-grant-access-to-user-andor-service-principal-at-root-scope--to-deploy-enterprise-scale-reference-implementation
